Source/ConversationV1/Models/MessageResponse.swift

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+/** A conversation response. */
+public struct MessageResponse: JSONDecodable {
+    
+    // Context, or state, from the Conversation system
+    public var context:  [String: JSON]?
+    
+    // Output, or response, from the system
+    public var output:   [String: JSON]?
```

Comment:
Instead of having type `[String: JSON]`, can we make this more descriptive? From the documentation it looks like the output has the following type:

```
inline_model_1 {
    text (string, optional)
}
```

What do you think about creating a `ConversationOutput` model or something similar to represent this type and mapping it from the JSON?